scylla-persistence/src/main/java/com/netflix/conductor/scylla/config/ScyllaProperties.java

```diff
@@ -48,7 +48,7 @@ public class ScyllaProperties {
     private int shardSize = 100;
 
     /** The replication strategy with which to configure the keyspace */
-    private String replicationStrategy = "SimpleStrategy";
+    private String replicationStrategy = "NetworkTopologyStrategy";
```

Comment:
We can keep the default to simple for local setup. and we can pass this from config for staging\prod?